cfgen/requirements.txt

```diff
@@ -1,2 +1,3 @@
+idna<3,>=2.5
```

Comment:
@knekrasov 
Do we need similar in file `provisioning\miniprov\requirements.txt`?